main.cpp

```diff
@@ -79,7 +107,7 @@ sort).\nPlease choose some numbers separated by commas! (ex. 1,2,3,4,5)\n";
 		return 0;
 	}
 	//create a bubblesort function
-	vector<double> sortedVector = bubbleSort(intNumberList);
+	vector<double> sortedVector = selectionSort(intNumberList);
```

Comment:
in the future we will need to produce a code for each and compare their efficiency.

Reply:
sure